src/tools/r2rdump/R2RDump.csproj

```diff
@@ -9,4 +9,9 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETCore.CoreDisTools" Version="1.0.1-prerelease-00003" />
+    <PackageReference Include="System.CommandLine" Version="0.1.0-e160119-1" />
```

Comment:
This used to be experimental command line parser. It is not actively maintained anymore and I doubt that it will ever ship stable version. You may want to ask around what the current best way to build command line parser is.